spec/controllers/collections_controller_spec.rb

```diff
@@ -71,6 +72,26 @@
         expect(doc[Solrizer.solr_name(:collection)]).to eq afterupdate.to_solr[Solrizer.solr_name(:collection)]
       end
     end
+
+    context "when setting visibility" do
+      it 'creates a public Collection' do
+        col1 = create(:collection, title: ["Public collection"],
```

Comment:
Can you use the `:public_collection` factory here? Then you can dump the explicit visibility.